helm-charts/nginx-ingress/templates/script-configmap.yaml

```diff
@@ -49,3 +49,26 @@ data:
 
     echo "Successfully created route ${NAME}."
     kubectl -n ${NAMESPACE} get route ${NAME}
+
+    kubectl -n ${NAMESPACE} get configmap ibmcloud-cluster-info
+    if [ $? -ne 0 ]; then
+        echo "Failed to get configmap ibmcloud-cluster-info, create it"
+        kubectl -n ${NAMESPACE} create configmap ibmcloud-cluster-info --from-literal=proxy_address=${HOST}
+        if [ $? -ne 0 ]; then
+            echo "Failed to create configmap ibmcloud-cluster-info"
+            exit 1
+        fi
+        kubectl label configmap ibmcloud-cluster-info1 app=management-ingress \
```

Comment:
typo for `ibmcloud-cluster-info1`.
You missed namespace in the command `-n ${NAMESPACE}`